manager/manager.go

```diff
@@ -208,3 +210,8 @@ func (m *NodeSetMockManager) UploadDepositDataToStakeWise(vaultAddress common.Ad
 func (m *NodeSetMockManager) MarkDepositDataSetUploaded(vaultAddress common.Address, network string, data []beacon.ExtendedDepositData) error {
 	return m.database.MarkDepositDataSetUploaded(vaultAddress, network, data)
 }
+
+// Call this once a deposit data set has been "registered" to StakeWise
+func (m *NodeSetMockManager) MarkValidatorRegistered(vaultAddress common.Address, network string, data []beacon.ExtendedDepositData) error {
```

Comment:
`MarkValidatorsRegistered` since there can be more than one, or `MarkDepositDataSetRegistered` to have parity with the uploaded function above.

Reply:
updated